site/src/pages.ts

```diff
@@ -16,6 +16,7 @@ export default [
     children: [
       { href: '/design-guidelines/accessibility', title: 'menu-accessibility' },
       { href: '/design-guidelines/color', title: 'menu-color' },
+      { href: '/design-guidelines/data-visualization/models', title: 'menu-data-visualization' },
```

Comment:
You should add the `menu-data-visualization` string entries to the locales at the `site/src/locales/` folder